packages/utils/src/index.ts

```diff
@@ -9,3 +9,4 @@ export { ECIES } from './ecies'
 export { IstanbulUtils } from './istanbul'
 export { PhoneNumberUtils } from './phoneNumbers'
 export { SignatureUtils } from './signatureUtils'
+export * from './string'
```

Comment:
Would be nice to wrap the exports to avoid polluting the namespace, like we do for the others here